src/org/starloco/locos/common/CryptManager.java

```diff
@@ -1,51 +1,48 @@
 package org.starloco.locos.common;
 
 import org.apache.commons.lang.StringEscapeUtils;
-import org.starloco.locos.area.map.GameCase;
-import org.starloco.locos.area.map.GameMap;
 
 import java.io.UnsupportedEncodingException;
 import java.net.URLDecoder;
-import java.util.*;
+import java.util.Arrays;
 
 public class CryptManager {
 
-    public final static char[] HASH = {'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p',
+    public final static char[] CELL_ENCODING_LOOKUP_TABLE = {'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p',
```

Comment:
This is just Base64 (with a slightly unconventional digits).  Let's just call it `BASE64_DIGITS`